tests/providers/amazon/aws/operators/test_redshift.py

```diff
@@ -42,3 +47,69 @@ def test_redshift_operator(self, test_autocommit, test_parameters, mock_get_hook
             autocommit=test_autocommit,
             parameters=test_parameters,
         )
+
+
+class TestResumeClusterOperator(unittest.TestCase):
+    @staticmethod
+    def _create_clusters():
+        client = boto3.client('redshift', region_name='us-east-1')
+        client.create_cluster(
+            ClusterIdentifier='test_cluster_to_pause',
+            NodeType='dc1.large',
+            MasterUsername='admin',
+            MasterUserPassword='mock_password',
+        )
+        client.create_cluster(
+            ClusterIdentifier='test_cluster_to_resume',
+            NodeType='dc1.large',
+            MasterUsername='admin',
+            MasterUserPassword='mock_password',
+        )
+        if not client.describe_clusters()['Clusters']:
+            raise ValueError('AWS not properly mocked')
+
+    def test_init(self):
+        redshift_operator = RedshiftResumeClusterOperator(
+            task_id="task_test", cluster_identifier="test_cluster", aws_conn_id="aws_conn_test"
+        )
+        assert redshift_operator.task_id == "task_test"
+        assert redshift_operator.cluster_identifier == "test_cluster"
+        assert redshift_operator.aws_conn_id == "aws_conn_test"
+
+    def test_resume_cluster(self):
```

Comment:
you can (and should) actually add some reasonable tests for this operator.

use standard mocking techniques

for the resume case you can at least verify that when the returned cluster state is `paused`, that `resume_cluster`  is called

similar for the pause case

Reply:
I would 100% want to add a unittest but due to the mock issue i describe here: https://github.com/apache/airflow/pull/19665#discussion_r759670233 and https://github.com/spulec/moto/issues/4591

So that's why I added the TODO until we are able to get `#4591` out.

Reply:
you are absolutely right @dstandish - i should have done more research on MagicMock and not give up just because `moto` didn't have this API implemented. 

I have been able to add unittests now using the strategy you suggested, THANK YOU SO MUCH!

Reply:
no worries at all, thank _you_ for your effort and contribution